single.php

```diff
@@ -7,24 +7,99 @@
 <main id="main-content" class="container">
 
   <!-- main posts loop -->
-  <section id="posts" class="row">
+  <section id="posts">
 
 <?php
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
 
+    $category = get_the_category( $post->ID );
+    $cat_name = $category[0]->cat_name;
+
     if ( has_category( 'featured' ) ) {
-      get_template_part( 'partials/single', 'featured' );
+      get_template_part( 'partials/featured-single', 'layout' );
     } else {
-      get_template_part( 'partials/single', 'post' );
+      get_template_part( 'partials/post', 'layout' );
     }
 
   }
 
+  // WP_Query arguments
+  $args = array (
+    'category_name'          => $cat_name,
+    'post__not_in'           => array($post->ID,),
+    'posts_per_page'         => '9',
+  );
+
+  // The Query
+  $query = new WP_Query( $args );
+
+  // The Loop
+  if ( $query->have_posts() ) {
+    $post_count = $query->found_posts;
+    $item_count = 1;
+?>
+
+    <div class="row">
+
+      <?php echo 'Más ' . $cat_name; ?> 
```

Comment:
i dont know what to wrap it in.  i think the style is too close to the 'ver mas' button from the front page.